### PR TITLE
Update changelog for release v44.15.0

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Updates the changelog for release v44.15.0. Also includes changes released on v44.11.0

### What issues does this PR fix or reference?

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+
+### Fixed
+
+#### salesforcedx-vscode-core
+
+- Enable multi-line SOQL selection for `SFDX: Execute SOQL Query with Currently Selected Text` ([PR #833](https://github.com/forcedotcom/salesforcedx-vscode/pull/833/files), [Issue #816](https://github.com/forcedotcom/salesforcedx-vscode/issues/816)). Contribution by [@boxfoot](https://github.com/boxfoot).
+
+- Fix syntax hightlighting for manifest XML files ([PR #823](https://github.com/forcedotcom/salesforcedx-vscode/pull/823))
+
+## 44.11.0 - December
```
Comment (ruthemmanuelle) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Shouldn't this include a specific date?

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Yes, good catch, it should be December 13, 2018

Comment (ruthemmanuelle) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Cool, I'll fix this when I make proofreading edits.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]: Thanks

**Comment by lcampos** [comment]
@ruthemmanuelle looks great, thanks for the edits 🙌

**Pull request merged**